tools/PI/DevHome.PI/Helpers/ExternalTool.cs

```diff
@@ -44,50 +45,36 @@ public string OtherArgs
     {
         get; private set;
     }
-
-    [JsonIgnore]
+
+    [ObservableProperty]
+    private bool _isPinned;
+
+    // Note the additional "property:" syntax to ensure the JsonIgnore is propagated to the generated property.
+    [ObservableProperty]
+    [property: JsonIgnore]
     private SoftwareBitmapSource? _toolIcon;
```

Comment:
>_toolIcon [](http://example.com/codeflow?start=34&length=9)

nit: can we use camelCase?

Reply:
Yes that would be my preferred style, but I'm seeing the leading underscore in other parts of DevHome. We should make an explicit style choice across the board, and do a code cleanup operation at some point.